pkgs/development/python-modules/flask-expects-json/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, flask
+, jsonschema
+}:
+
+buildPythonPackage rec {
+  pname = "flask-expects-json";
+  version = "1.7.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-TvGGqG8QVyohr4LlSVRt7aAkMmYotKls0U06D3F1TWI=";
+  };
+
+  propagatedBuildInputs = [ flask jsonschema ];
+
+  # tests aren't included in sdist and don't pass on current flask
```

Comment:
Then fetch git instead, because we will want to run those eventually.

Reply:
Done.